lib/gtfs/components/GtfsFilter.js

```diff
@@ -14,15 +14,14 @@ export default class GtfsFilter extends Component {
   }
 
   getFeedSummary (feeds) {
+    const feedNames = feeds.map(feed => feed.name.length > 11
+      ? feed.name.substr(0, 11) + '...'
+      : feed.name).join(' and ')
```

Comment:
Let's try to make this more readable by putting the ternary, the closing parenthesis and join call all on new lines.